### DISCUSSION TITLE
Projects/Organizations using Wagmi

### DISCUSSION THREAD

**Discussion by tmm** [wevm/wagmi#201] [Show and tell]
If you're using wagmi on your project/organization (company, side project, etc.), share the URL and/or GitHub link here.

**Comment by tmm** [comment]
midwit - https://midwit.vercel.app

**Reply by amit231** [reply]
repo?

**Comment by 0xDarni** [comment]
We are using it for an upcoming project at [Vector37 DAO](https://vector37.com) 🙂

**Comment by shunkakinoki** [comment]
light - https://light.so

**Reply by meri-maki** [reply]
Hi! saw your app and decided to check it out. And I couldn;t connect to ANY wallet via WalletConnect.... Nothing happens on screen and a lot of errors in console

**Comment by Dhaiwat10** [comment]
We are using it for our new frontend at Mazury -> https://github.com/mazurylabs/mazury-frontend

**Comment by peterferguson** [comment]
[Roundtable](https://getroundtable.xyz) - Mobile dApp for Investing with Friends on Avalanche (Polygon coming soon) 

🕸 [Website](https://getroundtable.xyz)
🐤 [Twitter](https://twitter.com/_getroundtable)

**Comment by Dhaiwat10** [comment]
`npx create-web3-frontend` -> https://github.com/Dhaiwat10/create-web3-frontend

**Comment by Hussainzz** [comment]
https://dev.to/hussainzz/eth-lottery-solidity-hardhat-3ebg

⚡⚡⚡⚡⚡   https://eth-lottery-3glp1l5k0-hussainzz.vercel.app/  ⚡⚡⚡⚡⚡

https://github.com/Hussainzz/Eth-Lottery

**Comment by gosseti** [comment]
[Foundation](https://foundation.app/)

**Comment by Nemusonaneko** [comment]
🦙 
https://twitter.com/llamapay_io
https://llamapay.io/

**Comment by nicoglennon** [comment]
Rainbow 🌈  - https://rainbow.me

**Comment by mdarche** [comment]
One of None - https://oneofnone.io

**Comment by straightupjac** [comment]
Referents :) [referents.xyz](https://referents.xyz/)

**Comment by ghost** [comment]
nfty chat 🙏  [nftychat.xyz](url)

**Comment by tmm** [comment]
[moonbirds.xyz](https://moonbirds.xyz)

https://twitter.com/ianhunter/status/1516183299785428996

**Reply by meri-maki** [reply]
Hi! Do you have a repo to look at? I wonder how to make steps for wallet connect (first connect - then ask to sign)

**Comment by matthewphiong** [comment]
[Blockvatar](https://blockvatar.com) - Free, globally unique CC0 NFT avatar for everyone.

We switched to wagmi from web3-react and it has been amazing. Thank you!

**Comment by codingwithmanny** [comment]
Thought I'd share this here before I finish off the written tutorial.
Tutorial on SIWE using wagmi with a backend node api in a monorepo. 

**Quick Vite React TypeScript Sign-In With Ethereum**
https://github.com/codingwithmanny/quick-vite-react-ts-siwe

**Reply by codingwithmanny** [reply]
Just updated this with `wagmi@0.3.1` support :)

**Comment by e-roy** [comment]
[create-web3](https://www.npmjs.com/package/create-web3) - `npx create-web3`  - a boilerplate for creating web3 projects, with nextjs, hardhat, and wagmi.   

❤️ the new v0.3.0

**Comment by sasicodes** [comment]
https://github.com/sasicodes/next-eth - boilerplate uses wagmi 🚀

**Comment by real-lazy-coder** [comment]
I have a branch on [Vector Finance](https://vectorfinance.io) that will switch over to using wagmi.  We hope to have it implemented in the coming weeks.

**Comment by matthewlilley** [comment]
Sushi 2.0 🍣

https://github.com/sushiswap/sushiswap

**Comment by delightfulabyss** [comment]
We're using WAGMI for one of our products over at [CabinDAO](https://www.creatorcabins.com/)!

**Comment by JorisZierold** [comment]
Using wagmi❤️  for [Rock Paper Scissors as a Gaming DApp](https://github.com/JorisZierold/gaming-dapp-rock-paper-scissors).

**Comment by blokboy** [comment]
💙 Using wagmi for [bbyDAO](https://bbydao.xyz) 💙

**Comment by davidlee1435** [comment]
[OpenGuild](https://openguild.finance) is using it!

**Comment by cesargdm** [comment]
An [alternate ENS app](https://github.com/cesargdm/ens-app) 😇

**Comment by wobsoriano** [comment]
A WIP Vue port of wagmi!

https://vagmi.vercel.app

**Comment by brendanjryan** [comment]
[Stripe](https://stripe.com/)

**Comment by Javier-Szyfer** [comment]
[lyingflat.place](https://lyingflat.place) wagmi hooks everywhere

**Comment by aramalipoor** [comment]
[flair.finance](https://flair.finance) We fully rely on `wagmi` for our NFT/Staking platform and our the open-source SDK 🚀

**Comment by JustinCast** [comment]
https://beta.everipedia.org/ 🚀🚀🛠

**Comment by linchengzzz** [comment]
P12 Genesis Soul-Bound NFT Airdrop, for gamers on Steam.

https://airdrop.p12.games/?code=dnHhSeyE
https://github.com/ProjectTwelve/airdrop-interface

**Comment by github-Dheeraj** [comment]
We have used WAGMI for Connection, transaction for Polygon, Mainnet, Arbitrum and optimism many more to come.
https://www.truts.xyz/
https://github.com/Daoverse123/client

**Reply by jitendra1731** [reply]
Jitendra singh

**Comment by 0xcadams** [comment]
Livepeer!
https://explorer.livepeer.org/

**Comment by shandysulen** [comment]
PopRank, an NFT rarity and aesthetic ranking app :)
https://poprank.io/

**Comment by 0xCarpeDiem** [comment]
First Omnichannel Collection that you can actively build and merge the traits of your NFT
https://myheropetz.com

**Comment by ghost** [comment]
A decentralised community for development and cybersecurity, built on Avalanche.

https://audicity.io/
https://app.audicity.io/

**Comment by naomiHauret** [comment]
jokedao V2 uses wagmi under the hood !
jokedao is a bottom-up, on-chain governance platform for communities to submit entries and vote on favorites: instead of central teams putting proposals to their community, communities vote on their own ideas.

https://github.com/JokeDAO/JokeDaoV2Dev/tree/main/packages/react-app-revamp
https://jokedao.io

**Comment by clague17** [comment]
spirals climat impact staking :) [spirals](https://www.spirals.so/) and [app](https://app.spirals.so/)

**Comment by jnebab** [comment]
using it at [rosehub.io](https://rosehub.io/). repo is private so can't share if that's ok :)

**Comment by seeARMS** [comment]
We heavily rely on wagmi at https://paragraph.xyz.

**Comment by josemvcerqueira** [comment]
Interest Protocol:

Github: https://github.com/interest-protocol/interface
Website: https://www.interestprotocol.com/

**Comment by rsodre** [comment]
Using wagmi for **Endless Crawler**, an infinite on-chain composable game.

https://endlesscrawler.io/

**Comment by sherrbss** [comment]
[Polymarket](https://polymarket.com/)

**Comment by nfts2me** [comment]
NFTs2Me -> [https://nfts2me.com/](https://nfts2me.com/)

🐤 [Twitter](https://twitter.com/nfts2me)

**Comment by zhyd1997** [comment]
Sponsor: https://github.com/zhyd1997/sponsor

**Comment by LilFatFrank** [comment]
Cruize -> https://beta.app.cruize.finance

**Comment by unholypanda** [comment]
Blacksmith is an adaptive local interface for Foundry projects

Wouldn't have been possible without wagmi

https://github.com/blacksmith-eth/blacksmith

https://twitter.com/0xholypanda/status/1598754870122643480

**Comment by wslyvh** [comment]
Created a new Next.js + Ethereum starter kit using wagmi

https://github.com/wslyvh/nexth/

**Comment by nas** [comment]
https://hashleap.io - We are simplifying crypto payments for small businesses, web3 companies, DAOs, freelancers and charities. We are making it easer to raise invoices, collect payments, record keeping while taking away the hassle and friction between each party.

We use Wagmi heavily alongwith WalletConnect V2

https://merchant.hashleap.io

Some example pages (Ofcourse, DO NOT MAKE ANY PAYMENTS as these pages are on mainnets): 
* A variable payment: https://dogfooding-ltd.hashleap.io/pay/checkout/6ws1f44sj1
* Fixed Payment: https://dogfooding-ltd.hashleap.io/pay/checkout/p93jnpy9uc
* Invoice Payment: https://dogfooding-ltd.hashleap.io/pay/invoice/HL-NEQC2Z-1002

**Comment by aj-may** [comment]
UseSIWE - The easiest way to add Sign-in with Ethereum to your app.
https://github.com/random-bits-studio/use-siwe

rainbowkit-use-siwe-auth - a [RainbowKit](https://www.rainbowkit.com/) authentication adapter for [UseSIWE](https://github.com/random-bits-studio/use-siwe).
https://github.com/random-bits-studio/rainbowkit-use-siwe-auth

**Comment by sunopar** [comment]
OnePunch: https://www.onepunch.finance/

**Comment by StKostyantin** [comment]
All-in-one Command Center for DeFi: -[https://q.xyz/](https://q.xyz/)

Track your historical portfolio value, access advanced data insights, and invest in DeFi protocols - all from a single place.

**Comment by clague17** [comment]
Currently using it to refactor https://syndicate.io 🚀

**Comment by smartydev42** [comment]
using on moonchan.io specifically [https://staking.moonchan.io](https://staking.moonchan.io)

**Comment by sternhenri** [comment]
We heavily work on wagmi here at [Privy](privy.io), both in our core work, and as a [core integration](https://docs.privy.io/guide/guides/wagmi) for Privy (here's [a good example](https://github.com/privy-io/wagmi-demo)).

**Comment by gauthamzz** [comment]
we use wagmi in [Polynomial](https://www.polynomial.fi/). https://trade.polynomial.fi/

**Comment by bigint** [comment]
Lenster - https://github.com/lensterxyz/lenster

**Comment by firnprotocol** [comment]
Firn has officially gone live with wagmi. LFG [app.firn.cash](https://app.firn.cash)

**Comment by Bot80926** [comment]
I create a web3 template use wagmi  which is called [web3-frontend-template](https://github.com/Bot80926/web3-frontend-template), aims to help frontend engineer quickly create a dapp. 
Template support function:

- Wallet connect (use wagmi)
- MultiChain 
- Auto refresh wallet balances
- Different Theme 
- I18n

Welcome to 🌟 / issue / fork ...

**Comment by joanseg** [comment]
At https://sprintcheckout.com we use wagmi!
We are the crypto payment processor focused on Ethereum rollups and Account Abstraction.

**Comment by johnpaulcas** [comment]
[web3transfer.xyz](https://www.web3transfer.io/#ether)

Inspired by disperse.app, This Dapp facilitates efficient distribution of multiple token (ERC20, ERC721, ERC1155) and native network coins - across EVM-compatible networks. By simplifying multi-recipient transfers

**Comment by xenide** [comment]
https://reservoir.fi 

Capital efficient AMM with classic and stable curves that allocates underutilized assets to money markets for yields. Earn yield on top of swap fees while you LP!

**Comment by unicape** [comment]
[use-wagmi](https://github.com/unicape/use-wagmi)
Vue Composition for Ethereum based on wagmi

**Comment by BritvaBo** [comment]
so. how many links

**Comment by oxSaturn** [comment]
We at Velocimeter use the amazing wagmi and viem on all our Dapps!

1. https://www.fvm.exchange/liquidity
2. https://base.velocimeter.xyz/liquidity
3. https://mantle.velocimeter.xyz/liquidity
4. https://cantov3.velocimeter.xyz/liquidity
5. https://pulse.velocimeter.xyz/liquidity

**Reply by hammeiam** [reply]
Is any of your frontend code open source?

**Reply by oxSaturn** [reply]
Hi, our repository was open sourced before, but closed recently. However you can still find some forks https://github.com/munj1/frontend_solidly/forks.

**Comment by tuler** [comment]
I created a new CLI plugin that integrates with `hardhat-deploy` artifacts.
https://github.com/sunodo/wagmi-plugin-hardhat-deploy

**Reply by tmm** [reply]
Super cool! Feel free to [create a new discussion](https://github.com/wagmi-dev/wagmi/discussions) specifically for this too.

**Reply by tuler** [reply]
https://github.com/wagmi-dev/wagmi/discussions/2947

**Comment by Yuripetusko** [comment]
We use wagmi and viem heavily on [Singular NFT marketplace](https://singular.app/l)

**Comment by kristiegiles** [comment]
https://enclave.market (app is https://trade.enclave.market)
We are a new crypto exchange built within an enclave to prevent front-running and data leakage, and we are using wagmi for web3 auth and connected wallet deposits.

**Comment by einaralex** [comment]
Monerium, regulated electronic Euro issuer.

www.monerium.com
www.monerium.app

 🚀

**Comment by meri-maki** [comment]
Just released the Web3 Wrapped 2023, made fully with Wagmi and viem

https://toon.org/wrapped

**Comment by chybisov** [comment]
Recently at [LI.FI](https://li.fi/), we released [LI.FI SDK v3](https://www.npmjs.com/package/@lifi/sdk) and [LI.FI Widget v3](https://www.npmjs.com/package/@lifi/widget) with Viem and Wagmi support.

https://github.com/lifinance/widget
https://github.com/lifinance/sdk
